wizard/views.py

```diff
@@ -2,6 +2,7 @@
 
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib import messages
```

Comment:
👍  in the future we should probably avoid that module (not very popular), but if it's already used here that's fine.